README.md

```diff
@@ -1822,13 +1829,14 @@ regarding stock awards and stock options.
   If it’s been years since the stock was first granted and the company is now worth a lot,
   the taxes owed could be quite significant.
 - 🄳 The Internal Revenue Code, in
-  [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers an alternative, called
+  [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers protection to anyone
```

Comment:
... offers taxpayers receiving equity in exchange for work the option to pay taxes early, before they vest, and thereby avoiding taxes at time of vesting. If qualified, a person can tell the IRS they prefer this alternative in a process called an [**83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/). Paying taxes early with an 83(b) election can potentially reduce taxes significantly. If the shares go up in value, the taxes owed at vesting might be far greater than the taxes owed at the time of receipt.

(found the previous wording a bit hard to parse)

Reply:
Clear change. Revised slightly for further clarity.